### PR TITLE
Release v0.0.34

### PR DESCRIPTION
**Pull request by justbuchanan**

### CODE REVIEW

**Comment by justbuchanan** [comment]
Release draft: https://github.com/kythe/kythe/releases/tag/untagged-14ccfa2dcea0e5783229

File: RELEASES.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Release Notes
 
+## [v0.0.34] - 2019-11-08
+
+#### Bug Fixes
+
+* **java extractor:**  Handle source files in jars (#4188) ([df711e31](https://github.com/kythe/kythe/commit/df711e3158718a923e11ecc86e51ab4e261ddfb5), closes [#4186](https://github.com/kythe/kythe/issues/4186))
+* **jvm extractor:**
+  *  use class files as srcs, not jar (#4191) ([907857c7](https://github.com/kythe/kythe/commit/907857c775341728505ab7dc67bcad8112901c02))
```
Comment (salguarnieri) [inline RELEASES.md]: nit: here and below move the single bullet inline like above

File: RELEASES.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+  *  don't count compilation units by corpus (#4195) ([88f5db4b](https://github.com/kythe/kythe/commit/88f5db4b54ba9482596dcabcb71f860f7a09fd8c))
+
+#### Features
+
+* **bazel extractor:**  put everything in the same corpus by default (#4192) ([3966e3f7](https://github.com/kythe/kythe/commit/3966e3f7aa790a61645e6babf511f7ae21b78011))
+* **go_indexer:**  option to emit an edge for an anchor's semantic scope (#4187) ([738f5fab](https://github.com/kythe/kythe/commit/738f5fabb15cd3fa6b31cfb7347e6456d13f05aa))
+* **java extractor:**  Include system modules in the compilation unit (#4194) ([b4b1b975](https://github.com/kythe/kythe/commit/b4b1b9759cff964cacc7bd0c825c264c7b1bc42b))
+* **java_indexer:**  add `generates` edge to generated AutoValue sources (#4193) ([f7614b19](https://github.com/kythe/kythe/commit/f7614b19e4384c97e076821d07b4fd7f3eacf389))
+* **runextractor:**  pipe underlying cmake stderr to stderr (#4178) ([aacea997](https://github.com/kythe/kythe/commit/aacea9970071ab4d7952daeab54bdf48c1d5ca2b))
+* **schema:**  support the generates relation between file nodes (#4177) ([ec5330b9](https://github.com/kythe/kythe/commit/ec5330b9c6f4f4d989b354277eeb2b95c3c3cb69))
```
Comment (salguarnieri) [inline RELEASES.md]: You can probably leave this out since the schema doc goes live outside of the normal release process

**Comment by salguarnieri** [comment]
Can you update the release draft also?

**Comment by justbuchanan** [comment]
Just updated the release.

**Pull request merged**